src/pages/journal/journal.ts

```diff
@@ -67,6 +78,9 @@ export class JournalPage extends BasePageComponent implements OnInit {
   start = '2018-12-10T08:10:00+00:00';
   merged$: Observable<void | number>;
   todaysDate: DateTime;
+  searchResults: SearchResultTestSchema[] = [];
+  searchResultsAppRefs: number[];
+  cachedTests: boolean;
```

Comment:
This prop could have a better name, perhaps `isCachedTests` as it is a boolean?

Reply:
`isCachedTests` is no longer present